react/features/display-name/components/web/styles.ts

```diff
@@ -46,26 +47,31 @@ export function getStageParticipantLineHeightRange(theme: Theme) {
  *
  * @param {Theme} theme - The current theme.
  * @param {number} clientHeight - The height of the visible area.
- * @returns {number}
+ * @returns {string}
  */
-export function getStageParticipantNameLabelHeight(theme: Theme, clientHeight?: number) {
-    const lineHeight = getStageParticipantNameLabelLineHeight(theme, clientHeight);
+export function getStageParticipantNameLabelHeight(theme: Theme, clientHeight?: number): string {
+    const lineHeight = parseFloat(getStageParticipantNameLabelLineHeight(theme, clientHeight));
```

Comment:
Can't we make the `getStageParticipantNameLabelLineHeight` return the rem value as a number to avoid `parseFloat`?